lib/shared/log/toConsole.js

```diff
@@ -14,6 +14,9 @@ var levels = [
 ];
 
 function toConsole(log, opts) {
+  // Remove previous listeners to enable to call this twice.
+  log.removeAllListeners();
```

Comment:
I think this should be more specific to avoid removing other listeners someone else might have attached (we are using a global, after all).  I can change this in the merge.